suite/tests/CMakeLists.txt

```diff
@@ -3749,6 +3751,13 @@ if (BUILD_CLIENTS)
 
     torunonly_drcacheoff(opcode_mix ${ci_shared_app} ""
       "@-simulator_type@opcode_mix" "")
+    # Ensure the tool works without the raw/ subdir.
+    set(${testname_full}_postcmd
```

Comment:
Does this really work? testname_full is defined inside torunonly_drcacheoff. Are we relying on that? Seems a bit hacky.

Reply:
Oh, this is a mistake: better to be the precise name `tool.drcacheoff.opcode_mix_postcmd`.  Fixed, here and the 2 others below.

Reply:
But yes it happened to work b/c the macro had set that var (macros don't have a separate namespace scope, unlike functions, so the var is visible).